### ISSUE TITLE
Improperly detects nil, as block

### ISSUE THREAD

**Issue by envygeeks** [thbar/kiba#70]
Given

```ruby
Kiba.run(Kiba.parse do
  source MySource
  destination MyDestination
  transform nil
end)
```

Kiba will throw an error that "Class and block form cannot be used together at the moment", but nil is not a block, it should throw a unique error that it found neither a block, or transformer.

**Comment by thbar** [comment]
Thanks Jordon! I will improve that!

Envoyé de mon iPhone

**Comment by thbar** [comment]
@envygeeks thanks again for the input ; I fixed this in #73 (which will be release into kiba 2.5), and will work at improving parse-time error messages in the future.

Many thanks!

**Comment by envygeeks** [comment]
Thanks!
